cmd/collector/collector.go

```diff
@@ -170,6 +186,9 @@ func run() error {
 	// Load the IPFIX global registry
 	registry.LoadRegistry()
 	// Initialize collecting process
+	if !slices.Contains(validDecodingModes, decodingMode) {
```

Comment:
I think we can delete
```
	if decodingMode == "" {
		decodingMode = DecodingModeStrict
	}
```
in initCollectingProcess

Reply:
I don't think so. `pkg/collector` is used as a library, including by Antrea. It makes sense to have a defaulting mechanism there.
`cmd/collector` is just a reference implementation used for testing & development.